src/11_6_2021_RcppClock_benchmarking_Rcpp_code.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "Benchmarking Rcpp code with RcppClock"
+author: "Zach DeBruine"
+license: GPL (>= 2)
+tags: Rcpp benchmarking RcppClock
+summary: Rcpp benchmarking with a simple C++ tick-tock clock and seamless integration
+  with the R environment at runtime.

```

Comment:
Seamless is a little overloaded with our project title, book title, and so on.  It's fine your package description, but maybe consider changing it here.  "direct"  maybe ?

Reply:
using the word "direct" instead of "seamless"